### DISCUSSION TITLE
Change cmap with just add_mesh_threshhold

### DISCUSSION THREAD

**Discussion by lucasgoddamn** [pyvista/pyvista#3368] [Q&A] [answered]
Hello,
so I'm trying to compare to MRIs by putting them next to each other and showing the differences in a heatmap. If i use the add_mesh function additionally i get a huge cube around that doesn't show anything and the colour remains default on the actual brain. Is there a way to get the same image as in subplot(0,0) with a threshhold slider but with the cmap="hot" (add_mesh_threshhold does not have a cmap arguement)? 
The subplot(1,0) just gives me the cube with heat colors around the brain and thats not what i want.
Maybe im missing a simple solution but thank you already in advance!

My code:

```py
plotter1 = pv.Plotter(shape=(2,2))

plotter1.subplot(0,0)
plotter1.add_text("Render Window 0", font_size=10)
plotter1.add_mesh_threshold(meshi, title="Threshhold")

plotter1.subplot(0,1)
plotter1.add_text("Render Window 1", font_size=10)
plotter1.add_mesh_threshold(meshi2, title="Threshhold")

plotter1.subplot(1,0)
plotter1.add_text("Render Window 2", font_size=10)
plotter1.add_mesh(meshi3, cmap="hot")
plotter1.add_mesh_threshold(meshi3, title="Threshhold")

plotter1.link_views()
plotter1.show()
```

**Comment by MatthewFlamm** [comment, marked as answer]
Does this do what you want?

```py
import pyvista as pv
mesh = pv.Sphere()
mesh["x"] = mesh.points[:, 0]
plotter = pv.Plotter()
plotter.add_mesh_threshold(mesh, cmap="hot")
plotter.show()
```

![image](https://user-images.githubusercontent.com/39341281/192605954-3aff5bbe-368a-4029-b152-e6b21caf7fd6.png)

**Reply by MatthewFlamm** [reply]
This method https://docs.pyvista.org/api/plotting/_autosummary/pyvista.Plotter.add_mesh_threshold.html passes extra kwargs to `add_mesh`.

**Reply by lucasgoddamn** [reply]
Yes exactly, for some reason i always got an exception that cmap is not a valid arg for add_mesh_threshhold but now it seems to work, thank you!
